scripts/translations_util.py

```diff
@@ -1,6 +1,36 @@
-from typing import List
+import os
 import subprocess
-from pylib.file.file_utils import FileUtils
+from typing import List
+
+
+def find_src_root() -> str:
+    '''Return the project source root directory.'''
+    dir = os.getcwd()
+    while (
+        dir
+        and dir != '/'
+        and os.path.isdir(dir)
+        and not os.path.exists(os.path.join(dir, '.git'))
+    ):
+        dir = os.path.dirname(dir)
+    return dir
+
+
+def get_absolute_filepath(filename: str) -> str:
```

Comment:
NITPICK: Typing can be updated here, you can pass in None and return None according to the logic.

```
Optional[str]
```